src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -916,6 +920,31 @@ public void parse(InputFile inputFile, Format ext, int type, boolean thumbOnly,
 		}
 	}
 
+	private static void addMusicBrainzIDs(AudioFile af, File file, DLNAMediaAudio currentAudioTrack) {
+		try {
+			Tag t = af.getTag();
+			if (t != null) {
+				String val = t.getFirst(FieldKey.MUSICBRAINZ_RELEASEID);
+				currentAudioTrack.setMbidRecord(val.equals("") ? null : val);
+				val = t.getFirst(FieldKey.MUSICBRAINZ_TRACK_ID);
+				currentAudioTrack.setMbidTrack(val.equals("") ? null : val);
+			}
+		} catch (Exception e) {
+			LOGGER.trace("audio musicBrainz tag not parsed: " + e.getMessage());
+		}
+	}
+
+	private static void addRating(AudioFile af, File file, DLNAMediaAudio currentAudioTrack) {
+		try {
+			Tag t = af.getTag();
+			if (t != null) {
+				currentAudioTrack.setRating(StarRating.convertTagRatingToStar(t));
+			}
+		} catch (Exception e) {
+			LOGGER.trace("audio rating tag not parsed: " + e.getMessage());
+		}
+	}
+
```

Comment:
These look very similar to the functions in libmediainfoparser, do these need to be here?

Reply:
It's even doubled in the same method ... I'll remove it ... thx for this one. 

But: I actually don't know why LibmediaInfo is parsing all the metadata (into the database ?), and then again DLNAMediaInfo.parse() doesn't use it. Probably because database access (cache ?) can be turned off.